pkgs/sketch_pad/lib/main.dart

```diff
@@ -143,8 +143,6 @@ class _DartPadAppState extends State<DartPadApp> {
             ColorScheme.fromSeed(seedColor: lightPrimaryColor).copyWith(
           surface: lightSurfaceColor,
           onSurface: Colors.black,
-          // TODO: Migrate to expanded Material 3 color palette.
-          // ignore: deprecated_member_use
```

Comment:
We need these ignores for now to keep the CI green -

Reply:
do I need to wait , until this ignore part is resolved by some other PR ?